language/dotnet/build-images.md

```diff
@@ -0,0 +1,307 @@
+---
+title: "Build your .NET image"
+keywords: .net, build, images, dockerfile
+description: Learn how to build your first Docker image by writing a Dockerfile
+---
+
+{% include_relative nav.html selected="1" %}
+
+## Prerequisites
+
+Work through the orientation and setup in Get started [Part 1](../../get-started/index.md) to understand Docker concepts.
+
+{% include guides/enable-buildkit.md %}
```

Comment:
Maybe for a later PR. Nowadays buildkit is enabled by default in current Docker Desktop versions (IIRC it's default with Docker 20.10.x), so we probably don't need this include anymore.

Reply:
Thanks, I'll remove it. It looks like it was recently removed from the other guides as well.